helpers/securewebhook/securewebhook.go

```diff
@@ -1,14 +1,25 @@
 package securewebhook
 
 import (
+	"crypto/ecdsa"
+	"crypto/sha256"
+	"encoding/asn1"
+	"encoding/base64"
 	"encoding/json"
+	"math/big"
 )
 
 // Settings ...
 type Settings struct {
 	Enable *bool `json:"enabled,omitempty"`
 }
 
+// RS ...
```

Comment:
for clarity purposes, could you explain what RS means?

Reply:
added